.github/workflows/on_release.yaml

```diff
@@ -0,0 +1,44 @@
+name: Deploy to NPM
+on:
+  release:
+    types: [published]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+
+      - name: Install packages
+        if: success() && startsWith(github.ref, 'refs/tags/v')
+        run: |
+          npm install
+
+      - name: Build and deploy
+        if: success() && startsWith(github.ref, 'refs/tags/v')
+        env:
+          GITHUB_REF_TAG: ${{ github.ref }}
+          NPMJS_REGISTRY: registry.npmjs.org
+          NPMJS_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          export VERSION=${GITHUB_REF_TAG:10:50}
```

Comment:
Can't find it in the docs how the `github.ref` looks like - is the version part always in the same place in the string (does the string change (apart from the version part of it)?

Reply:
It always starts with `refs/tags/`, though I'm also having trouble locating appropriate docs.

I agree it is not obvious what is happening... I have changed the extraction of `VERSION` a bit, do you think this is better?

Reply:
Yes, I think it's better now.
Resolving.